distro/run/assembly/resources/start.bat

```diff
@@ -1,108 +1,27 @@
-) ELSE (
-  IF NOT EXIST "%JAVA_HOME%" (
-    ECHO JAVA_HOME "%JAVA_HOME%" path doesn't exist
-    GOTO :EOF
-  ) ELSE (
-    IF NOT EXIST "%JAVA_HOME%\bin\java.exe" (
-      ECHO "%JAVA_HOME%\bin\java.exe" does not exist
-      GOTO :EOF
-    )
-    ECHO Setting JAVA property to "%JAVA_HOME%\bin\java"
-    SET JAVA="%JAVA_HOME%\bin\java"
-  )
-)
-
-IF NOT "x%JAVA_OPTS%" == "x" (
-  ECHO JAVA_OPTS: %JAVA_OPTS%
-)
-
-REM set environment parameters
-SET webappsPath=%BASEDIR%internal\webapps
-SET restPath=%BASEDIR%internal\rest
-SET swaggerPath=%BASEDIR%internal\swaggerui
-SET examplePath=%BASEDIR%internal\example
-SET classPath=%BASEDIR%configuration\userlib,%BASEDIR%configuration\keystore
-SET optionalComponentChosen=false
-SET restChosen=false
-SET swaggeruiChosen=false
-SET productionChosen=false
-SET configuration=%BASEDIR%configuration\default.yml
-
-
-REM inspect arguments
-:Loop
-IF [%~1]==[] GOTO Continue
-
-IF [%~1]==[--webapps] (
-  SET optionalComponentChosen=true
-  SET classPath=%webappsPath%,%classPath%
-  ECHO WebApps enabled
-) 
-
-IF [%~1]==[--rest] (
-  SET optionalComponentChosen=true
-  SET restChosen=true
-  SET classPath=%restPath%,%classPath%
-  ECHO REST API enabled
-)
-
-IF [%~1]==[--swaggerui] (
-  SET optionalComponentChosen=true
-  SET swaggeruiChosen=true
-  SET classPath=%swaggerPath%,%classPath%
-  ECHO Swagger UI enabled
-)
-
-IF [%~1]==[--example] (
-  SET optionalComponentChosen=true
-  SET classPath=%examplePath%,%classPath%
-  ECHO Invoice Example included - needs to be enabled in application configuration as well
-)
-
-IF [%~1]==[--production] (
-  SET productionChosen=true
-  SET configuration=%BASEDIR%configuration\production.yml
-)
+REM Start detached if no arguments
+IF [%~1]==[] GOTO StartDetached
 
+REM Collect arguments to pass to the executable
+SET CMD_LINE_ARGS=
+:setArgs
+IF [%~1]==[] GOTO StartWithArguments
+SET CMD_LINE_ARGS=%CMD_LINE_ARGS% %1
 SHIFT
-GOTO Loop
-:Continue
-
-REM If no optional component is chosen, enable REST and Webapps.
-REM If production mode is not chosen, also enable Swagger UI and the example application.
-setlocal enabledelayedexpansion
-IF [%optionalComponentChosen%]==[false] (
-  SET restChosen=true
-  ECHO REST API enabled
-  ECHO WebApps enabled
-  IF [%productionChosen%]==[false] (
-    SET swaggeruiChosen=true
-    ECHO Swagger UI enabled
-    ECHO Invoice Example included - needs to be enabled in application configuration as well
-    SET classPath=%swaggerPath%,%examplePath%,%classPath%
-  )
-  SET classPath=%webappsPath%,%restPath%,!classPath!
-)
-setlocal disabledelayedexpansion
+GOTO setArgs
 
-REM if Swagger UI is enabled but REST is not, warn the user
-IF [%swaggeruiChosen%]==[true] (
-  IF [%restChosen%]==[false] (
-    ECHO You did not enable the REST API. Swagger UI will not be able to send any requests to this Camunda Platform Run instance.
-  )
-)
+:StartDetached
+call "%EXECUTABLE%" start --detached
+REM open a browser
+timeout /t 10 /nobreak > NUL
```

Comment:
@mboskamp @tmetzke I changed the `ping` command to `timout` since it's similar to the `sleep` command in SHELL.

Reply:
That works better for me. In the end, it's a matter of timing but at least for me the browser did not show the "not responding" error page again. 👍